controls/os_spec.rb

```diff
@@ -21,7 +21,7 @@
 login_defs_passmaxdays = attribute('login_defs_passmaxdays', default: '60', description: 'Default password maxdays to set in login.defs')
 login_defs_passmindays = attribute('login_defs_passmindays', default: '7', description: 'Default password mindays to set in login.defs')
 login_defs_passwarnage = attribute('login_defs_passwarnage', default: '7', description: 'Default password warnage (days) to set in login.defs')
-if os.redhat?
+if os.redhat? || os.suse?
   shadow_group = 'root'
 elsif os.debian?
   shadow_group = 'shadow'
```

Comment:
@chris-rock @atomic111 we do not have here a sane default. It means in the situation prior to this change on a suse system shadow_group would be `nil`. How should we handle this? Options from my POV:

1. introduction of else and raise, something like:
```ruby
shadow_group = if os.redhat?
                              'root'
                            elsif
                              'shadow'
                            else
                              raise 'Unsupported platform'
                            end
```

2. defined sane default, like

```ruby
shadow_group = 'root'
if os_debian?
  shadow_group = 'shadow'
end
```

Ideas, opinions?